src/panes.ts

```diff
@@ -172,6 +172,7 @@ export type FinishedPane = Pane<
       display_name: string
     }[]
     total_device_count?: number
+    is_reconnect?: boolean
```

Comment:
maybe
```suggestion
    was_account_reconnected?: boolean
```
if I'm understanding it correctly?

Reply:
yes, makes more sense